hail/python/hailtop/utils/tqdm.py

```diff
@@ -1,10 +1,8 @@
-from tqdm.notebook import tqdm as tqdm_notebook
-from tqdm.auto import tqdm as tqdm_auto
-
-# To tqdm_notebook, None means do not display. To standard tqdm, None means
-# display only when connected to a TTY.
-TQDM_DEFAULT_DISABLE = False if tqdm_auto == tqdm_notebook else None
-
-
-def tqdm(*args, disable=TQDM_DEFAULT_DISABLE, **kwargs):
+def tqdm(*args, disable=False, **kwargs):
+    from tqdm.notebook import tqdm as tqdm_notebook  # pylint: disable=import-outside-toplevel
+    from tqdm.auto import tqdm as tqdm_auto  # pylint: disable=import-outside-toplevel
+    # To tqdm_notebook, None means do not display. To standard tqdm, None means
+    # display only when connected to a TTY.
+    if disable is False and tqdm_auto != tqdm_notebook:
```

Comment:
I think this is technically semantically different as it means you cannot pass `disable=False` to standard tqdm…

Reply:
Maybe this is fine, or else I introduce some other `disable` value that means "whatever the default disable is"

Reply:
I prefer that we do not change the semantics of disable. Use a unique distinct value for the default value of the disable parameter. Like:
```
DEFAULT_DISABLE = object()
```